### DISCUSSION TITLE
Klipperscreen wont flip after trying to install Crowsnest and picam v3

### DISCUSSION THREAD

**Discussion by codiac2600** [KlipperScreen/KlipperScreen#1243] [Q&A]
Some crazy reason I cannot flip klipperscreen anymore after installing crows nest and a pi camera module 3. Can't find any way to fix it either.

**Comment by alfrix** [comment]
all the information that i know about rotation is here:
https://klipperscreen.github.io/KlipperScreen/Troubleshooting/Rotation/
